### ISSUE TITLE
TypeError: Cannot read property 'url' of undefined

### ISSUE THREAD

**Issue by davidhellmann** [himynameisdave/git-labelmaker#70]
```
TypeError: Cannot read property 'url' of undefined
 at module.exports (/usr/local/lib/node_modules/git-labelmaker/bin/modules/readRepo.js:13:40)
 at Promise.all.then (/usr/local/lib/node_modules/git-labelmaker/bin/index.js:38:19)
```

I've a repo with dots like `com.foobar.www` maybe that's a Problem. 
Any Idea how I can fix that?

Node 7.4.0

**Comment by himynameisdave** [comment]
I believe other people have also experienced this issue. Personally I haven't tried it on Node 7+ (but I sort of doubt thats the issue). 

I will look more into this, thanks for reporting!

**Comment by himynameisdave** [comment]
@davidhellmann what exactly was your Git url? In your `.git/config` file, did it look something like this?
```
[remote "origin"]
	url = git@www.github.com:davidhellmann/repo-name.git
```

If so, I don't think that test case is being covered in the [`node-github-url-from-git`](https://github.com/tj/node-github-url-from-git/blob/master/test.js) package we use to parse the git config. 

Please respond with the exact git url that gave you trouble and I will file & fix in the appropriate place 👍 😄

**Comment by himynameisdave** [comment]
@davidhellmann re: https://github.com/himynameisdave/git-labelmaker/issues/70#issuecomment-287567458 do you have a git url for me? If not I may defer this fix until a later release so that I can get `0.9.0` released and out to users when it's stable

**Comment by himynameisdave** [comment]
@davidhellmann any chance you can update me [re: my comment above ☝️](https://github.com/himynameisdave/git-labelmaker/issues/70#issuecomment-290810024)

**Comment by himynameisdave** [comment]
Actually I believe this issue was fixed when #66 was fixed (d46aea62502e2a7c542710dbd94174762a51e7d3).

Closing, but will reopen if you are still experiencing the issue.
